vagrant/provisioning/roles/alfresco7/tasks/main.yml

```diff
@@ -405,31 +405,31 @@
       become: yes
       become_user: alfresco7
       stat:
-        path: "{{ root_folder }}/app/alfresco7/shared/classes/alfresco/extension/ce-log4j.properties"
+        path: "{{ root_folder }}/app/alfresco7/shared/classes/alfresco/extension/ce-{{ log4j_properties }}"
```

Comment:
This {{ log4j_properties }} need to be stated somewhere with a value or default if not provided one. Maybe i am missing where that is in this PR?

Reply:
The value is provided from the facts.yml. it uses log4j2.properties instead of log4j.properties now should I add the default value as log4j.properties

Reply:
> The value is provided from the facts.yml. it uses log4j2.properties instead of log4j.properties now should I add the default value as log4j.properties

That facts.yml file is provided by you when you run AWS provision, so we don't have that values here. Provide them as well in the facts files for this projects.